app/src/main/res/xml/scene_event_details.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This file looks like it needs reformatting, all tag closings are on a newline by themselves and it looks odd

Reply:
Reformatting doesn't change any line tho (I made sure I have the Project code style settings)

Reply:
It may be because the xml resources are not considered "Android" XML and so can have a different style? At any rate, could you fix those? And maybe open an issue for AS :)

Reply:
👍 for enforcing it for me